config/routes.rb

```diff
@@ -1,15 +1,19 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
   root "users#index"
 
-  # get "/users", to: "users#index", as: "users"
-  # get "/users/:id", to: "users#show", as: "user"
-  # get "/users/:user_id/posts", to:"posts#index", as: "user_posts"
-  # get "/users/:user_id/posts/:id", to: "posts#show", as: "user_post"
-  resources :users, only:[:index, :show] do
-    resources :posts, only: [:index, :show]
-  end
+  get "/users/:id", to: "users#show", as: "user"
+  get "/users", to: "users#index", as: "users"
+
+  post "/users/:user_id/posts/:id/newcomment", to: "comments#create", as: "create_new_comment"
```

Comment:
- [ ] Great job with your implementation of the comments. However, per the project requirements, there should be a pair of routes in the `config/routes.rb` file for creating comments similar to what you did for `Posts create` and `Posts New`. Currently, you only have the `Post` route for comments in your project :+1: 
- Example
   ```ruby
      POST   /posts/:post_id/comments(.:format)     comments#create
    GET   /posts/:post_id/comments/new(.:format)     comments#new
   ```
- [ ] After doing this, you should ensure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments. :+1:

Reply:
create comment is under post controller

Reply:
well resolved! ✔️